smslibrary/src/main/java/com/eis/smslibrary/SMSReceivedBroadcastReceiver.java

```diff
@@ -6,15 +6,22 @@
 import android.os.Build;
 import android.os.Bundle;
 import android.telephony.SmsMessage;
+import android.util.Log;
+
+import it.lucacrema.preferences.PreferencesManager;
```

Comment:
The package's name is `com.eis.preferences`

Reply:
Thanks, I had to edit the package name because I didn't want to keep my name on it, but the compiler forgot to tell me that I had to change on the smslibrary files too